.circleci/config.yml

```diff
@@ -122,8 +122,9 @@ jobs:
   prep-build:
     docker:
       - image: circleci/node@sha256:e16740707de2ebed45c05d507f33ef204902349c7356d720610b5ec6a35d3d88
+    resource_class: medium+
```

Comment:
What was the reason for increasing this? Was this just to make the intermittent failures less likely, or did something introduced in this PR affect these jobs?

Reply:
this helped alleviate CI failures. I think we run 3x parallel node processes with 2gig "max old space" => `medium+` for 6GB mem

Reply:
we could try removing this now that we're no longer parsing and transforming ses lockdown dist

Reply:
`prep-build` & `prep-build-test` was consistently failing with out of memory. These steps from other PRs seemed to be passing just fine, so it must have been something from here that caused it. I'm not sure what though.